### DISCUSSION TITLE
Binding data to Objects

### DISCUSSION THREAD

**Discussion by NeloReis** [Budibase/budibase#2774] [Help]
<img width="1387" alt="Screen Shot 2021-09-28 at 14 06 38" src="https://user-images.githubusercontent.com/5040389/135134837-6c2330ef-decd-46cb-a499-307c1e9ec78d.png">

Noob Question Here.
I'm trying to create a semi-custom score card.
to display the result of sums or any other math.
is there a more elegant way (fewer elements) to achieve this?
the 3 elements---  data provider -> table(hidden) -> binding element
is the only way I know how to pass data to an element.
I think there must be a better way to display results.?
am I missing something? 
thanks!

**Comment by mjashanks** [comment]
Try using a 

```
> Data Provider
    > Repeater
        > Container
```

![image](https://user-images.githubusercontent.com/3524181/135159294-574f551b-06ae-4c4b-a738-9bc19a39ffc2.png)

**Reply by NeloReis** [reply]
thank you.. that helps a lot. Now ill be moving on to the math problems. 
can't wait to get my hands on themes e JS.. an awesome project. 
I truly believe from what is see this could be the WordPress of apps.
